src/hotspot/os/linux/crac_linux.cpp

```diff
@@ -462,3 +463,7 @@ bool crac::read_bootid(char *dest) {
   }
   return true;
 }
+
+void crac::initialize_time_counters() {
+  os::Posix::init();
```

Comment:
The initialization updates some condition variable attributes; have you checked if there's any chance that another  thread could observe an inconsistent state while this is updated?

Reply:
You're right, I missed it.